From 38fb7f3bc94e1fd6196df4a0482ed00b646c1e8e Mon Sep 17 00:00:00 2001
From: Alex Nitz <alex.nitz@gmail.com>
Date: Tue, 22 Feb 2022 15:19:25 +0100
Subject: [PATCH] Switch to minimal pegasus api requirement (#3943)

* Switch to minimal pegasus api requirement

This is an attempt to address the immediate issue of Pegasus having a hard requirement on flask which no longer works due to #3942. In general, it's probably a good idea to keep our dependencies as lean as possible as well.

Assuming this passes tests, is there any reason we don't *just* depend on the API? Doe we actually need anything else for the PyCBC library directly?

* forgot to change here too

* I guess I'll have to do it this way...
---
 requirements.txt | 5 +++++
 setup.py         | 2 +-
 2 files changed, 6 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 4003c755676..a1ae9589a77 100755
--- a/setup.py
+++ b/setup.py
@@ -54,7 +54,7 @@
                       'ligo-segments',
                       'tqdm',
                       'gwdatafind',
-                      'pegasus-wms >= 5.0.1',
+                      'pegasus-wms.api >= 5.0.1',
                       'python-ligo-lw >= 1.7.0'
                       ]
 
